ui/static/ui/js/utils.jsx

```diff
@@ -125,16 +125,17 @@ define("utils", ["jquery", "lodash", "react", "react_infinite", "select2"],
   /**
    * Generic confirmation dialog.
    * <ConfirmationDialog
-   *       id={confirmationDialogId}
```

Comment:
In general we don't need to set the id attribute for the confirmation dialog, we already have a reference to it and we can style it using CSS classes.